core/src/test/java/org/elasticsearch/index/seqno/GlobalCheckpointTrackerTests.java

```diff
@@ -117,6 +108,11 @@ public void testGlobalCheckpointUpdate() {
         // it is however nice not to assume this on this level and check we do the right thing.
         final long minLocalCheckpoint = allocations.values().stream().min(Long::compare).orElse(UNASSIGNED_SEQ_NO);
 
+        final GlobalCheckpointTracker tracker = new GlobalCheckpointTracker(
```

Comment:
maybe add a helper method `newTracker(String allocationId)`, so that there is less code repetition.

Reply:
I pushed e69b8d993fe99b57422f8466a391099d159c4681.